README.md

```diff
@@ -1,76 +1,54 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
 
-> One paragraph statement about the project.
+
+> Using linters
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
-## Built With
+#Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- VS Code
 
-## Live Demo
+Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Getting Started
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+Prerequisites
+You only need a browser
 
-## Authors
 
-👤 **Author1**
+Authors
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 Author1
 
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/iLynette)
+- Twitter: [@twitterhandle](https://twitter.com/acholah_lynette)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/lynette-acholah/)
```

Comment:
Kindly customize the README.md file to your project. Kindly remove the second author's details.